apps/backend/src/bookings/bookings.service.ts

```diff
@@ -47,6 +49,24 @@ export class BookingsService implements IBookingsService {
     return results;
   }
 
+  async getBookingsByGuideId(
+    guideId: number,
+  ): Promise<GetBookingsByGuideIdResponse> {
+    const bookings = await this.bookingsRepo.paginateBookings({
+      offset: 0,
+      limit: 100000,
```

Comment:
please use value from frontend input, too much limit will make database burden

Reply:
OK offset and limit added as queryparams